### PR TITLE
Adds two new mugs

### PR DESCRIPTION
**Pull request by Partheo**
Adds a mug with an american flag to the brig office, and a mug with a canadian flag to the cargo office.

deh9uvb
7nvflzh

This PR was made to illustrate my GitKraken guide at: https://forums.yogstation.net/index.php?threads/release-the-gitkraken-how-to-make-your-first-pull-request.15099/

:cl: Partheo
tweak: The british cup in the medbay is now a mug
rscadd: Added a new american mug to security
rscadd: Added a new canadian mug to cargo
/:cl:

### CODE REVIEW

**Comment by 1fbff5f83b23d39d38b1dfcb4cac8d9b** [comment]
you made a typo in the changelog, in the second line: rcsadd instead of rscadd. also you are missing /:cl:

**Comment by Partheo** [comment]
Yeah, I'm aware. I did it so that I can get an image of a failed changelog check. Doesn't seem like that's a thing anymore though.

**Comment by Partheo** [comment]
What the hell

File: code/modules/jobs/job_types/civilian.dm [hunk 1]
```diff
@@ -1,6 +1,8 @@
 /*
 Clown
 */
+(break bitch(
```
Comment (Time-Green) [inline code/modules/jobs/job_types/civilian.dm]: I think this is causing the error

Comment (Partheo) [reply code/modules/jobs/job_types/civilian.dm]: Thank you

**Comment by Partheo** [comment]
I put coffee in the mugs

File: _maps/map_files/Yogstation/yogstation.2.1.3.dmm [hunk 2]
```diff
@@ -1100,6 +1100,11 @@
 /area/ai_monitored/security/armory)
 "acj" = (
 /obj/structure/table/reinforced,
+/obj/item/weapon/reagent_containers/food/drinks/mug/amerimug{
+	desc = "A mug with the american flag emblazoned on it. You can almost taste the freedom.";
```
Comment (AshCorr) [inline _maps/map_files/Yogstation/yogstation.2.1.3.dmm:1104]: Why do you set the desc here? Surely this desc could be used for all american mugs?

File: code/modules/food_and_drinks/drinks/drinks.dm [hunk 3]
```diff
@@ -170,13 +170,16 @@
 	icon_state = "tea"
 	item_state = "coffee"
 	spillable = 1
+	var/staticicon = FALSE //If true, sprite does not update and remains the same
 
 /obj/item/weapon/reagent_containers/food/drinks/mug/on_reagent_change()
-	if(reagents.total_volume)
-		icon_state = "tea"
+	if(staticicon == TRUE)
```
Comment (AshCorr) [inline code/modules/food_and_drinks/drinks/drinks.dm:176]: Perhaps `if(!staticicon)` and then get rid of the else....

File: code/modules/jobs/job_types/civilian.dm [hunk 4]
```diff
@@ -1,6 +1,7 @@
 /*
 Clown
 */
+
```
Comment (AshCorr) [inline code/modules/jobs/job_types/civilian.dm:4]: GAME BREAKING CHANGE THERE

File: _maps/map_files/Yogstation/yogstation.2.1.3.dmm (hunk 2 above)
Comment (Partheo) [reply _maps/map_files/Yogstation/yogstation.2.1.3.dmm:1104]: This is the only american mug

File: code/modules/jobs/job_types/civilian.dm (hunk 4 above)
Comment (Partheo) [reply code/modules/jobs/job_types/civilian.dm:4]: It's all part of my secret plan

File: code/modules/food_and_drinks/drinks/drinks.dm (hunk 3 above)
Comment (Partheo) [reply code/modules/food_and_drinks/drinks/drinks.dm:176]: I'm dumb

**Pull request closed without merging**